wormhole-connect/src/views/v2/Bridge/Routes/index.tsx

```diff
@@ -165,7 +131,7 @@ const Routes = ({ ...props }: Props) => {
       {props.isLoading && renderRoutes.length === 0 ? (
         <Skeleton variant="rounded" height={153} width="100%" />
```

Comment:
Nitpick: Instead of one empty skeleton, do you think we can have two small horizontal ones at the top and a bigger one below those, to imitate this look:

<img width="441" alt="Screenshot 2024-10-24 at 3 59 25 PM" src="https://github.com/user-attachments/assets/5ac36bc0-c924-45d7-bca1-3a4e7301e997">

Reply:
probably but I'd rather do that in a separate PR.